crates/swc_ecma_minifier/examples/minifier.rs

```diff
@@ -58,7 +58,7 @@ fn main() {
         )
         .expect_module();
 
-        let output = output.fold_with(&mut fixer(None));
+        let output = output.fold_with(&mut hygiene()).fold_with(&mut fixer(None));
```

Comment:
Please revery this

Reply:
So it should only apply fixer but not `hygiene`? I would like to understand why.